whats-new-panel.yaml

```diff
@@ -113,22 +35,59 @@
   targeting: firefoxVersion >= 78
   trigger:
     id: whatsNewPanelOpened
-- id: WHATS_NEW_BADGE_78
+- id: WHATS_NEW_BADGE_80
   content:
     action:
       id: show-whatsnew-button
     badgeDescription:
       string_id: cfr-badge-reader-label-newfeature
-    bucket_id: WHATS_NEW_BADGE_78
+    bucket_id: WHATS_NEW_BADGE_80
     delay: 300000
     target: whats-new-menu-button
   frequency:
     lifetime: 100
   priority: 5
-  targeting: firefoxVersion == 78 && (usesFirefoxSync || hasAccessedFxAPanel || currentDate|date
-    - 2419200000 >= profileAgeCreated) && (!messageImpressions['WHATS_NEW_BADGE_78']
-    || (messageImpressions['WHATS_NEW_BADGE_78']|length >= 1 && currentDate|date -
-    messageImpressions['WHATS_NEW_BADGE_78'][0] <= 345600000)) && 'browser.messaging-system.whatsNewPanel.enabled'|preferenceValue
+  targeting: firefoxVersion == 80 && (usesFirefoxSync || hasAccessedFxAPanel || currentDate|date
+    - 2419200000 >= profileAgeCreated) && (!messageImpressions['WHATS_NEW_BADGE_80']
```

Comment:
Hmm, I am not sure if those filters `(usesFirefoxSync || hasAccessedFxAPanel || currentDate|date
    - 2419200000 >= profileAgeCreated)` are still relevant to 80, could you confirm?

Reply:
Added comment.